### PR TITLE
Updated return structs, made all tests pass, added bundler tasks

### PR DESCRIPTION
**Pull request by ejhayes**
This pull gets the tests to pass again (missing continent and icon from the result structs).  Also updated the version number and add bundler default tasks (so that you can do rake install and rake build).  It also looks like test_search was defined twice, so I consolidated that into a single test.

Thanks for writing this gem!

### CODE REVIEW

File: Rakefile [hunk 1]
```diff
@@ -1,4 +1,5 @@
 require 'rake/testtask'
+require 'bundler/gem_tasks'
```
Comment (kalmbach) [inline Rakefile:2]: It's a matter of taste, is not a requirement. I use the gem commands.

File: lib/osmn/structs.rb [hunk 2]
```diff
@@ -11,14 +11,16 @@ module OSMN
     :class,
     :type,
     :importance,
-    :address)
+    :address,
+    :icon)
```
Comment (kalmbach) [inline lib/osmn/structs.rb:15]: Can't find any reference in the Nominatim API to this field.

File: test/test_osmn.rb [hunk 3]
```diff
@@ -15,12 +15,6 @@ def test_nil_reverse
     assert_equal(nil, response)
   end
 
-  def test_search
-    response = OSMN::search('135 pilkington avenue, birmingham')[0]
-
-    assert_equal('62311100', response.place_id)
-  end
-
```
Comment (kalmbach) [inline test/test_osmn.rb:23]: Is not actually a duplicated test.
The other test verifies the result of the search function when searching with no details.

Searching with no details is the default behaviour, we have tests for the default behaviour and for when specifically requesting a search with no details.

**Comment by kalmbach** [comment]
Closing pull request. However, I took the Address struct change and the duplicated test name to create a couple of commits that fix both of them. Thanks for your observations.

File: test/test_osmn.rb (hunk 3 above)
Comment (ejhayes) [inline test/test_osmn.rb:23]: True, I see the different functionality, but the function name can't be used twice.  Looks like you addressed this though :)

**Comment by ejhayes** [comment]
Thanks!

**Pull request closed without merging**